manager/container.go

```diff
@@ -304,7 +305,7 @@ func (c *containerData) housekeeping() {
 		default:
 			// Perform housekeeping.
 			start := time.Now()
-			c.housekeepingTick()
+			nextCollectionTime = c.housekeepingTick()
```

Comment:
I think we don't need to change the housekeeping time. Supporting collection of custom metrics at a rate faster than regular housekeeping will be problematic. 

How about making UpdateCustomStats() equivalent to UpdateStats(), and calling both of these from tick. We can save the next time for customMetric collection in containerData.